helper-cli/src/main/kotlin/commands/ListLicenseCategoriesCommand.kt

```diff
@@ -90,14 +90,6 @@ class ListLicenseCategoriesCommand : CliktCommand(
 
     private fun License.description(ignoreCategory: String? = null): String {
         val categories = categories.toMutableList().apply {
-            if (includeInNoticeFile) {
-                add("include-in-notices")
```

Comment:
Also, I believe the remaining code block could be simplified to just `categories.remove(ignoreCategory)` as `categories` is a `SortedSet` which already is mutable, and removing `null` would do no harm.

Reply:
I don't think that the categories of the License should actually be modified when generating a description; but maybe the code could be simplified to something like:
`val categories = categories.filterNot { it == ignoreCategory }`

However, this brings up an important topic: IMHO, the set of categories in _License_ should not be mutable. Should we change it to a Set instead of a SortedSet? The order of categories is probably of interest only when something is displayed to the user.

Reply:
> I don't think that the categories of the License should actually be modified when generating a description

I agree, and your proposed solution sound feasible to me, expect that we should not shadow the `categories` variable.

> IMHO, the set of categories in License should not be mutable. Should we change it to a Set instead of a SortedSet?

I also agree here. Historically, we use (Java's) `SortedSet` instead of (Kotlin's) `Set` in too may places where sorting is not really required, and might even harm performance (e.g. on insertion into large sets).-